pxe/common_files/disk_imaging.sh

```
@@ -890,6 +1018,13 @@ function main() {
 	if [ -z "$image_disk" -o "$image_disk" == "smallest" ]; then
 		find_bootdisk
 	fi
+	resolved_image_disk=$(readlink -e -- "$image_disk")
+	if [ -z "$resolved_image_disk" ] || [ ! -b "$resolved_image_disk" ]; then
+		echo "Image disk $image_disk does not exist or is not a block device" | tee $log_output
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)